ariadne_ai/llms/rag/answer_relevance.py

```diff
@@ -64,34 +64,47 @@ class AnswerRelevance:
         1. Consider the following: 
         user's query: {}.
         response:{}.
-        2. Determine if the response answers specifically what the user is asking about, and covers all aspects of the user's query.
-        3. Provide a brief explanation of why the response does or does not answer the user's query sufficiently, labeled as 'explanation', leading up to a verdict (Yes/No) labeled as 'verdict'.
-        4. Return a JSON object in the following format: "verdict": 'verdict', "explanation": 'explanation'.
+        2. Make sure to also consider these instructions: {}
```

Comment:
If there are no additional instructions, this will be an empty string. 

This works, but is probably not ideal - we can fix this in the next release and remove the additional instruction altogether.